example/pico-ssram/firmware.c

```diff
@@ -12,10 +12,17 @@
 #define DATA_LEN 8
 #define START_ADDR 0
 
+static volatile int g_sequence_count;
+
+void async_callback(void) {
+    ++g_sequence_count;
```

Comment:
This is an useful API, I will try to use it more through the rest of the SDK when appliable.